dmc2gym/wrappers.py

```diff
@@ -37,42 +40,59 @@ def _flatten_obs(obs):
 class DMCWrapper(core.Env):
     def __init__(
         self,
+        # suite module environment loading function
+        suite_load_fn,
+        # env kwargs
         domain_name,
         task_name,
-        task_kwargs=None,
-        visualize_reward={},
+        task_kwargs={},
+        environment_kwargs=None,
+        visualize_reward=False,
+        # step_kwargs
+        frame_skip=1,
         from_pixels=False,
+        # render_kwargs
         height=84,
         width=84,
         camera_id=0,
-        frame_skip=1,
-        environment_kwargs=None,
-        channels_first=True
+        # observation_kwargs
+        channels_first=True,
     ):
-        assert 'random' in task_kwargs, 'please specify a seed, for deterministic behaviour'
         self._from_pixels = from_pixels
-        self._height = height
-        self._width = width
-        self._camera_id = camera_id
         self._frame_skip = frame_skip
         self._channels_first = channels_first
 
+        # set render kwargs
+        self.render_kwargs = dict(
+            height=height,
+            width=width,
+            camera_id=camera_id,
+        )
+
         # create task
-        self._env = suite.load(
+        self._env = suite_load_fn(
             domain_name=domain_name,
             task_name=task_name,
             task_kwargs=task_kwargs,
+            environment_kwargs=environment_kwargs,
             visualize_reward=visualize_reward,
-            environment_kwargs=environment_kwargs
         )
 
+        # setup metadata for rendering since dm_control bases on MuJoCo Physics
+        # options from gym.envs.mujoco.mujoco_env
+        self.metadata = {
+            "render.modes": ["human", "rgb_array", "grey", "notebook"],
+            # "video.frames_per_second": int(np.round(1.0 / self.dt)),
```

Comment:
Delete or uncomment?

Probably is nice to specify, dunno if `dm_control` exposes `dt` in an easily accessible way.

Reply:
Sure! Found the same trick in [gym.mujoco_env](https://github.com/openai/gym/blob/150404104afc57ea618acca34d2d1f797d779869/gym/envs/mujoco/mujoco_env.py#L62).